src/patternfly/components/Toolbar/toolbar.scss

```diff
@@ -2,16 +2,22 @@
 // @import "../../sass-utilities/all";
 $pf-c-toolbar--breakpoint-map: build-breakpoint-map("base", "md", "lg", "xl", "2xl");
 $pf-c-toolbar--spacer-map: build-spacer-map("none", "sm", "md", "lg");
+$pf-c-toolbar--inset-map: build-spacer-map();
```

Comment:
without values passed to the function, looks like it will build these values https://github.com/patternfly/patternfly/blob/4a1a56f1eb307fd25b87993687de9594069b5f20/src/patternfly/sass-utilities/scss-variables.scss#L302-L309

If so, the docs should be updated. They currently list `.pf-m-inset-{none, sm, md, lg, xl, 2xl}{-on-[breakpoint]}`